site/src/sphinx/server-decorator.rst

```diff
@@ -135,6 +135,45 @@ a certain type from a server:
         }
     }
 
+.. _server-decorator-service-with-path-mappings:
+
+Decorating ``ServiceWithPathMappings``
+--------------------------------------
+
+:api:`ServiceWithPathMappings` is a special variant of :api:`Service` which allows a user to register multiple
+routes for a single service easily. Because it requires a special treatment unlike usual :apiplural:`Service`,
+the two following restrictions apply:
+
+- You cannot specify path mappings when registering a :api:`ServiceWithPathMappings`.
```

Comment:
Might want to clarify that actually they can, but it won't use the `ServiceWithPathMappings` in such a case.

Reply:
Ah, yeah. Let me rephrase it a little bit.

Reply:
Updated. PTAL again :-)